ai/core/src/unitycatalog/ai/core/executor/common.py

```diff
@@ -0,0 +1,77 @@
+    "EXECUTOR_TIMEOUT. The default value is 20 seconds.\n"
+)
+
+NO_OUTPUT_MESSAGE = (
+    "The function execution has completed, but no output was produced. "
+    "This may indicate that the function did not return a value or that an error occurred "
+    "without being captured. Check the function implementation and ensure it returns a value "
+    "by locally executing the function returned from calling the `get_function_source` method on the function "
+    "client object and converting your function to a callable using `unitycatalog.ai.core.utils.execution_utils.load_function_from_string`."
+)
+
+OPEN_DISALLOWED_MESSAGE = (
+    "The use of the 'open' function is restricted within the local sandbox executor for safety reasons. "
+    "This is to prevent the execution of potentially dangerous code or access to sensitive system resources. "
+    "If you need to read or write files within your function, utilize a different execution mode in your function client."
+)
+
+IMPORT_DISALLOWED_MESSAGE = (
+    "You may not use this module within the local sandbox executor for safety reasons. "
+    "This is to prevent the execution of potentially dangerous code or access to sensitive system resources. "
+    "Set a different execution mode in your function client in order to execute this function."
+)
+
+
+def generate_terminated_message(signal_num: int) -> str:
+    """
+    Generate a message indicating that the process was terminated by a signal.
+
+    Parameters:
+        signal_num: The signal number that caused the termination.
+
+    Returns:
+        A string message indicating the termination reason.
+    """
+    return (
+        f"The function execution has been terminated with a signal {signal_num}. "
+        "This likely indicates that the function exceeded its resource limits "
+        "(CPU execution time or exceeded its virtual memory cgroup allocation). "
+        "You can adjust the following environment variables to increase the limits if "
+        "you are certain that your function is behaving as expected:\n"
+        "\t1. EXECUTOR_MAX_CPU_TIME_LIMIT: Maximum CPU execution time limit for the executor "
+        "measured in CPU seconds (total CPU execution time, not wall-clock time).\n"
+        "\t2. EXECUTOR_MAX_MEMORY_LIMIT: Maximum memory limit for the executor measured in MB. "
+        "The default value is 100 MB. Note that this restriction is only applicable to Linux environments "
+        "and has no functional use in Mac OSX or Windows.\n"
+    )
+
+
+def generate_import_disallowed_message(module_name: str) -> str:
+    """
+    Generate a message indicating that the import of a module is disallowed.
+
+    Parameters:
+        module_name: The name of the disallowed module.
+
+    Returns:
+        A string message indicating the disallowed import.
+    """
+    return f"The import of module '{module_name}' is restricted. {IMPORT_DISALLOWED_MESSAGE}"
```

Comment:
Seems we don't need a function here. We can define a string and use .format on module_name. Similar for generate_terminated_message